packages/compass-query-bar/src/utils/query.ts

```diff
@@ -149,6 +149,11 @@ export function validateField(
     }
   }
 
+  // HACK to validate hint document as a sort document
```

Comment:
I think that's totally okay and not even really a hack, I'd maybe just explain in this comment that we don't have a validator for indexes so the sort validator works fine as a replacement?

Reply:
Updated!